store/store.go

```diff
@@ -1976,14 +1976,21 @@ type CurrentSnap struct {
 	TrackingChannel  string
 	RefreshedDate    time.Time
 	IgnoreValidation bool
+	Epoch            snap.Epoch
 	Block            []snap.Revision
 }
 
+type epochJSON struct {
```

Comment:
did we store epochs in JSON previously?  I think we should simplify snap/epoch.go to not JSON serialize to the simplified forms instead or we will get it wrong at some point

Reply:
https://github.com/snapcore/snapd/pull/6130